tools/deb/istio-iptables.sh

```diff
@@ -1,78 +1,46 @@
 #!/bin/bash
 # Initialization script responsible for setting up port forwarding for Istio sidecar.
 
-set -o errexit
 set -o nounset
 set -o pipefail
 
-usage() {
-  echo "${0} -p ENVOY_PORT -u UID -s IN_SERVICE_PORTS [-h]"
-  echo ''
-  echo '  -p: Specify the envoy port to which redirect all TCP traffic'
-  echo '  -u: Comma separate list of UIDs for which the redirection is not'
-  echo '      applied. Typically, this is the UID of the proxy container'
-  echo '  -i: Comma separated list of IP ranges in CIDR form to redirect to envoy (optional)'
-  echo '  -s: Comma separated list of incoming service IPs to intercept (optional)'
-  echo ''
-}
-
-IP_RANGES_INCLUDE=""
-
-while getopts ":p:u:e:i:h:s" opt; do
-  case ${opt} in
-    p)
-      ENVOY_PORT=${OPTARG}
-      ;;
-    u)
-      ENVOY_UID=${OPTARG}
-      ;;
-    i)
-      IP_RANGES_INCLUDE=${OPTARG}
-      ;;
-    s)
-      ISTIO_LOCAL_PORTS=${OPTARG}
-      ;;
-    h)
-      usage
-      exit 0
-      ;;
-    \?)
-      echo "Invalid option: -$OPTARG" >&2
-      usage
-      exit 1
-      ;;
-  esac
-done
-
-if [[ -z "${ENVOY_PORT-}" ]]; then
-  ENVOY_PORT=15001
+ISTIO_SIDECAR_CONFIG=${ISTIO_SIDECAR_CONFIG:-/var/lib/istio/envoy/sidecar.env}
+if [ ! -r $ISTIO_SIDECAR_CONFIG ]; then
+  echo "Missing sidecar config: $ISTIO_SIDECAR_CONFIG"
+  exit 1
 fi
 
-if [[ -z "${ENVOY_UID-}" ]]; then
-  ENVOY_UID=`id -u ${ENVOY_USER-istio}`
+. $ISTIO_SIDECAR_CONFIG
+
+if [ -z "${ENVOY_UID:-}" ]; then
```

Comment:
you still have `:-}` here (does that change anything vs just the variable ? (I don't know))